core/src/main/java/org/testcontainers/containers/ContainerState.java

```diff
@@ -35,6 +37,17 @@ default Boolean isRunning() {
         }
     }
 
+    /**
+     * @return has the container health state 'healthy'?
+     */
+    default Boolean isHealthy() {
```

Comment:
unwrap to `boolean`

Reply:
Same as above, `isRunning()` did use `Boolean`, so I wanted to be consistent.